CharacterData.py

```diff
@@ -1,112 +1,190 @@
+			return self.__rates
+
+		if False:
+			"""
+			GrowthRates wrap a dictioanry of individual growth rates for a given
+				character/class combination
+
+			Attributes:
+				__rates: A dictionary mapping stats onto growth rates
+			"""
+			__rates = {}
+
+			def __init__(self, array):
+				"""Inits GrowthRates with rates"""
+				self.rates = {}
+				self.rates['HP'] = int(array[2])
+				self.rates['Str'] = int(array[3])
+				self.rates['Mag'] = int(array[4])
+				self.rates['Skl'] = int(array[5])
+				self.rates['Spd'] = int(array[6])
+				self.rates['Lck'] = int(array[7])
+				self.rates['Def'] = int(array[8])
+				self.rates['Res'] = int(array[9])
+				pass
+				
+			def __str__(self):
+				"""Creates string for GrowthRates object"""
+				rt_str = ''
+				for key in self.rates:
+					rt_str += '\t' + str(key) + ' : '
+					rt_str += str(self.rates[key]) + '\n'
+				return rt_str
+				
+			def getRates(self):
+				return self.rates
+
+
+	def __init__(self, name):
+		"""Inits a CharacterData object 
+
+		Args:
+			name : a str of the character's name
+
+		Returns:
+			CharcterData object
+		"""
+		pass
+
+	def add_initial_state(self, state):
+		"""Sets the initial state for CharacterData
+
+		Args:
+			state : State object for character's initial state
+		"""
+		pass
+
+	def get_growth_rate(self, game_class):
+		"""Retrieves GrowthRates object for given in game class
+
+		Args:
+			game_class : a str describing an in game class
+
+		Returns:
+			GrowthRates object associated with the desired in game class
+		"""
+		pass
+
+	def predict_state(self, level, game_class):
+		"""Creates a state for the average stats for a character with a given 
+			in game class and level
+
+		Args:
+			level : an int representing an in game level
+			game_class: a str describing an in game class
+
+		Returns:
+			A state object representing the average for this character at the
+				given class and level.
+		"""
+		pass
+
+	if False:
+		"""
+		CharacterData contains constant values set by the game for a character
+
+		Attributes:
+			name: A string describing a character's name
+			game_class_options: A list of strings describing the in game classes
+				a character can choose from
+			growth_rate_class: A dicitonary mapping class names (from 
+				game_clas_options) onto GrowthRates objects  
+			base_stats: A dictionary mapping stat names to the character's base class stats
+			base_class: The class this character is in when they join in the game
+		"""
 		def __init__(self, array):
-			"""Inits GrowthRates with rates"""
-			self.rates = {}
-			self.rates['HP'] = int(array[2])
-			self.rates['Str'] = int(array[3])
-			self.rates['Mag'] = int(array[4])
-			self.rates['Skl'] = int(array[5])
-			self.rates['Spd'] = int(array[6])
-			self.rates['Lck'] = int(array[7])
-			self.rates['Def'] = int(array[8])
-			self.rates['Res'] = int(array[9])
+			"""Inits CharacterData with name, game_class_options and 
+			growth_rate_class"""
+			self.name = array[0]
```

Comment:
Wrapped in an "if False" block. To be replaced by private variable __name described in "Attributes" portion of doc_string at the top of file. Resolved